cli/src/benchmark_gpt.ts

```diff
@@ -119,7 +119,8 @@ async function main(args: Required<CLIArguments>): Promise<void> {
     for (let i = 0; i < iterations; i++) {
       const timeStart = performance.now()
       for (let n = 0; n < maxNewTokens; n++) {
-        const next: number = (await model.predict(List.of(tokens))).first();
+				const next = (await model.predict(List.of(tokens))).first();
+				if (next === undefined) throw new Error("empty prediction");
```

Comment:
Is there a lot of whitespace here or is it github freaking out

Reply:
ha yes, I'm using tabs for indentation now via [biome](https://biomejs.dev/) which is shown as two spaces in my editor (but not github it seems)

shall we [commit to a formatter](https://github.com/epfml/disco/issues/631) one of theses days?

Reply:
Yes it might be time